classes/Deck.js

```diff
@@ -0,0 +1,665 @@
+
+
+export default class Deck {
+  constructor() {
+    this.cards = [
+      {
+        image: require("../components/cards/png/1.png"),
```

Comment:
Giving these card images clearer names would make it easier to find a specific card later - this one could be `solid_red_squiggle_1.png` for example.